config/locales/zh_CN.yml

```diff
@@ -0,0 +1,40 @@
+zh_CN:
+  errors:
+    messages:
+      taken_in_past: '曾被使用过。'
+      equal_to_current_password: '必须与当前密码不同。'
+      password_complexity:
+        digit:
+          one: 必须包含至少1个数字
+          other: 必须包含至少%{count}个数字
+        lower:
+          one: 必须包含至少1个小写字母
+          other: 必须包含至少%{count}个小写字母
+        symbol:
+          one: 必须包含至少1个标点符号
+          other: 必须包含至少%{count}个标点符号
+        upper:
+          one: 必须包含至少1个大写字母
+          other: 必须包含至少%{count}个大写字母
+  devise:
+    invalid_captcha: '无效验证码'
+    invalid_security_question: '无效安全问题答案'
+    paranoid_verify:
+      code_required: '请输入我们支持团队提供的代码'
+    paranoid_verification_code:
+      show:
+        submit_verification_code: 提交验证码
+        verification_code: 验证码
+        submit: 提交
+    password_expired:
+      updated: '新密码已保存'
+      change_required: '你的密码已过期，请更新密码.'
+      show:
+        renew_your_password: 更新你的密码
+        current_password: 当前密码
+        new_password: 新密码
+        new_password_confirmation: 确认新密码
+        change_my_password: 更改密码
+    failure:
+      session_limited: '你的登录凭据已在另一个浏览器上被使用。请重新登录以在此浏览器继续。'
```

Comment:
Is `。` the same as `.`? It looks like in some examples you use `.` and `。` in others like this.  Would it make sense to use `。` in all examples?

Reply:
Yes, `。` is the Chinese period. I will change them to `。` and rephrase things I removed the periods.